skeleton.html

```diff
@@ -8,29 +8,32 @@
     <title>Match Game</title>
   </head>
   <body>
-    <div>
-      <button id="1"></button>
-      <button id="2"></button>
-      <button id="3"></button>
-      <button id="4"></button>
+    <div class="flex-container">
+      <button class="gameSpace 1"></button>
```

Comment:
looks like according to the rules a CSS selector cannot start with a number. It must start with a `-`, `_` or a letter. 
It can have a number in it, just not at the start

https://www.w3.org/TR/CSS21/grammar.html#scanner